### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by Gamecock**
Provide example on customization based on guidance provided in #988.

### CODE REVIEW

File: contrib/opencensus-ext-django/README.rst [hunk 1]
```diff
@@ -40,6 +38,30 @@ for a complete reference.
             )''',
         }
     }
+    
+Additional configuration can be provided, please read
```
Comment (lzchen) [inline contrib/opencensus-ext-django/README.rst:42]: I would put these lines `Additional configuration...` at the end.

File: contrib/opencensus-ext-django/README.rst [hunk 2]
```diff
@@ -40,6 +38,30 @@ for a complete reference.
...
+`Customization <https://github.com/census-instrumentation/opencensus-python#customization>`_
+for a complete reference.
+
+For customization the exporter has to be instantiated separately
```
Comment (lzchen) [inline contrib/opencensus-ext-django/README.rst:46]: Technically this does not HAVE to be instantiated separately to work. The only reason why we need to pass in the instance is because we need to utilize the `add_telemetry_processor` api.

I would reword this to a more generic:

`For customization of the exporter: `

.. code:: python
    from opencensus.ext.azure.trace_exporter import AzureExporter
    exporter = AzureExporter(service_name='mysite')

    OPENCENSUS = {
        'TRACE': {
            'SAMPLER': 'opencensus.trace.samplers.ProbabilitySampler(rate=1)',
            'EXPORTER': exporter
        }
    }

File: contrib/opencensus-ext-django/README.rst [hunk 3]
```diff
@@ -40,6 +38,30 @@ for a complete reference.
...
+        'TRACE': {
+            'SAMPLER': 'opencensus.trace.samplers.ProbabilitySampler(rate=1)',
+            'EXPORTER': exporter
+        }
+    }
+    
+    def callback(envelope):
+    return False
+
+Both forms assumes Environmental Variable ``APPINSIGHTS_INSTRUMENTATIONKEY`` exists.
```
Comment (lzchen) [inline contrib/opencensus-ext-django/README.rst:64]: I would remove this line because `opencensus-ext-django` shouldn't have anything to do with `azure`.

File: contrib/opencensus-ext-django/README.rst (hunk 1 above)
Comment (Gamecock) [reply contrib/opencensus-ext-django/README.rst]: Done

File: contrib/opencensus-ext-django/README.rst (hunk 2 above)
Comment (Gamecock) [reply contrib/opencensus-ext-django/README.rst]: Not exactly sure what you are looking for here.  I left the line whit `add_telemetry_processor` because that is the part I was trying to explain.

File: contrib/opencensus-ext-django/README.rst (hunk 3 above)
Comment (Gamecock) [reply contrib/opencensus-ext-django/README.rst]: Done

**Review by Gamecock — COMMENTED** [review]
Addressed the comments, still working on the liscencing document.

**Comment by Gamecock** [comment]
@googlenot I signed it!

File: contrib/opencensus-ext-django/README.rst [hunk 4]
```diff
@@ -40,6 +38,28 @@ for a complete reference.
...
+
+    OPENCENSUS = {
+        'TRACE': {
+            'SAMPLER': 'opencensus.trace.samplers.ProbabilitySampler(rate=1)',
+            'EXPORTER': exporter
+        }
+    }
+    
+    def callback(envelope):
+    return False
```
Comment (lzchen) [inline contrib/opencensus-ext-django/README.rst:58]: ### 
```suggestion
        return False
```

**Review by lzchen — APPROVED** [review]

**Review by lzchen — APPROVED** [review]

File: contrib/opencensus-ext-django/README.rst (hunk 4 above)
Comment (Gamecock) [reply contrib/opencensus-ext-django/README.rst]: Done
